src/Translatable/Translatable.php

```diff
@@ -24,6 +24,8 @@ trait Translatable
 
     protected static $autoloadTranslations = null;
 
+    protected static $deleteTranslationsCascade = false;
```

Comment:
If it's a property on the trait we will need two setter methods like for the autoloading one. Otherwise you can only set this property on a class that extends the class using the package trait. That's why your unittest passes - otherwise you will get a property conflicts exception.

Reply:
Didn't know that! I've always managed this approach with a non-existent property by default 

```
if (property_exists(self::class, 'deleteTranslationsCascade') && self::$deleteTranslationsCascade) {
    ...
}
```

But this approach is more self-documented 👌

Reply:
We use this approach in spatie packages. It requires a `@property` annotation. But I would like to keep consistent with this package and it saves this condition. Both have (dis)advantages.